woodwork/tests/schema/test_schema.py

```diff
@@ -226,6 +253,9 @@ def test_filter_schema_cols_no_matches(sample_column_names, sample_inferred_logi
     filter_non_string = schema._filter_cols(include=1)
     assert filter_non_string == []
 
+    filter_exclude_all = schema._filter_cols(exclude=list(sample_inferred_logical_types.values()))
+    assert filter_exclude_all == []
```

Comment:
I think this might be equivalent to the check in the table accessor tests where we check 
`assert len(schema_df.ww.select(exclude=all_types).columns) == 0`

In this case "no matches" might be the `schema._filter_cols(exclude=[])` return the full list of columns as opposed to "everything matches" returning no columns

Reply:
Ah, I see. I was reading this initially as testing for an empty list being returned instead of "no columns found matching the specified criteria" which I now see is the intent. I'll update.